select/sdks/ios/migration-guide.md

```diff
@@ -36,6 +36,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
     
     private func configureFidel() {
+        // It is recommended to store the SDK key on your backend and retrieve it
+        // before starting to use the SDK.
```

Comment:
Would "opening" or "launching" the SDK work better here? "Starting to use" sounds a bit strange, because from a developer PoV, when I'm implementing the SDK that's when I start using it, but we want to retrieve the keys on runtime while opening it (at least on the Web SDK we use the concept of opening the SDK, don't know about the mobile one)

Reply:
You are right “starting to use” is not a good formulation. But also, “launching” or “opening” refers to starting a card enrollment flow, so I would decouple SDK “configuration” process from “launching" process. I would write the following:

`We recommend storing the SDK key on your server. Please use one of your endpoints to retrieve it and then configure the sdkKey property with it.`

Please let me know, if that works better.

Reply:
Yes, that does sound clearer!